File: CitizenDemo-DotNetCore/src/CitizenDemo.CitizenAPI/Models/Citizen.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace CitizenDemo.CitizenAPI.Models
{
    public class Citizen
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        [BsonElement("_id")]
        public string? InternalId { get; set; }

        [BsonElement("citizenId")]
        public string? CitizenId { get; set; }

        [BsonElement("givenName")]
        public string? GivenName { get; set; }

        [BsonElement("surname")]
        public string? Surname { get; set; }

        [BsonElement("phoneNumber")]
        public string? PhoneNumber { get; set; }

        [BsonElement("streetAddress")]
        public string? StreetAddress { get; set; }

        [BsonElement("city")]
        public string? City { get; set; }

        [BsonElement("state")]
        public string? State { get; set; }

        [BsonElement("postalCode")]
        public string? PostalCode { get; set; }

        [BsonElement("country")]
        public string? Country { get; set; }

        public override string ToString()
        {
            return string.Format(
                "Citizen: CitizenId: {0}, GivenName: {1}, Surname: {2}, PhoneNumber: {3}, " +
                "StreetAddress: {4}, City: {5}, State: {6}, PostalCode: {7}, Country: {8}", CitizenId,
                GivenName, Surname, PhoneNumber, StreetAddress, City, State, PostalCode, Country
                );
        }
    }
}

File: CitizenDemo-Java/src/citizenapi/src/main/java/com/citizendemo/citizenapi/models/Citizen.java
package com.citizendemo.citizenapi.models;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Document(collection = "citizens")
public class Citizen {
    @Id
    public String internalId;
    public String citizenId;
    public String givenName;
    public String surname;
    public String phoneNumber;
    public String streetAddress;
    public String city;
    public String state;
    public String postalCode;
    public String country;

    public Citizen() {
    }

    public Citizen(String citizenId, String givenName, String surname, String phoneNumber, String streetAddress, String city, String state, String postalCode, String country) {
        this.citizenId = citizenId;
        this.givenName = givenName;
        this.surname = surname;
        this.phoneNumber = phoneNumber;
        this.streetAddress = streetAddress;
        this.city = city;
        this.state = state;
        this.postalCode = postalCode;
        this.country = country;
    }

    @Override
    public String toString() {
        return "Citizen{" +
                "citizenId='" + citizenId + '\'' +
                "givenName='" + givenName + '\'' +
                "surname='" + surname + '\'' +
                "phoneNumber='" + phoneNumber + '\'' +
                "streetAddress='" + streetAddress + '\'' +
                "state='" + state + '\'' +
                "postalCode='" + postalCode + '\'' +
                "country='" + country + '\'' +
                '}';
    }
}